core/src/main/scala/kafka/coordinator/group/GroupMetadataManager.scala

```diff
@@ -1070,12 +1090,11 @@ object GroupMetadataManager {
    * @return payload for offset commit message
    */
   private[group] def offsetCommitValue(offsetAndMetadata: OffsetAndMetadata): Array[Byte] = {
-    // generate commit value with schema version 1
+    // generate commit value with schema version 2
     val value = new Struct(CURRENT_OFFSET_VALUE_SCHEMA)
```

Comment:
nit: from the original code, but it's kind of weird to refer to the "current" schema here, but use v2 (or v1) fields below.